File: LMS/LMS.UI/huda-center/src/app/shared/models/library-management/products/product-to-update.dto.ts
export interface ProductToUpdateDto {
  productId: string;
  arProductName: string;
  arProductDescription: string;
  enProductName: string;
  enProductDescription: string;
  productPrice: number;
  productStock: number;
  categoriesIds: string[]; 
  imgUrl: string;
}

File: LMS/LMS.Application/DTOs/LibraryManagement/Products/ProductToUpdateDto.cs
using LMS.Application.DTOs.LibraryManagement.Categories;

namespace LMS.Application.DTOs.LibraryManagement.Products
{
    public class ProductToUpdateDto
    {
        public string ARProductName { get; set; }
        public string ARProductDescription { get; set; }
        public string ENProductName { get; set; }
        public string ENProductDescription { get; set; }
        public decimal ProductPrice { get; set; }
        public int ProductStock { get; set; }
        public ICollection<CategoryLookUpDto> Categories { get; set; } = [];
        public string ImgUrl { get; set; } = string.Empty;
    }
}
